src/ToDo.UI/Views/TaskListPage.xaml

```diff
@@ -118,6 +119,8 @@
 														<utu:AutoLayout PrimaryAxisAlignment="End"
 																		Orientation="Horizontal">
 															<ToggleButton utu:AutoLayout.CounterAlignment="Start"
+																		  Command="{controls:AncestralBinding Path=StarTask, AncestorType=uer:FeedView}"
```

Comment:
cant bind to the ItemsControl here because its is wrapped in a FeedView, so the data-context is the feed thing
and not the page vm